ui/webui/src/components/storage/DiskEncryption.scss

```diff
@@ -0,0 +1,5 @@
+#installation-destination-next-spinner {
```

Comment:
So i tool a better look and realized this is not correct. Generally it's not adviced to add custom CSS, we should rely on patternfly paddings and use layout components from spacing. 

In this case, we should try to follow https://www.patternfly.org/v4/components/empty-state/design-guidelines/#empty-states-inside-tables-cards-or-wizards, so we want 24px padding on the top. 

The empty state component has default top padding 32 px, and the wizard main body has default padding 24 px. So we are currently, without your overrides and by removing the Bullseye at 56. It's more than double of the advised padding.

So if you want to be  100% correct, you keep the padding from the wizard body, which is 24px (1.5rem) and you reset to 0 the empty state component's padding.

Reply:
I feel like the 24px rule is generally meant for elements that are embedded within a larger page and since the wizard is full-screen in our case, I think the full-page rule fits it better.

Thanks for your patience, by the way.

Reply:
Sorry for taking long time to reply. I don't have strong opinion between the in-page vs wizard, indeed that's a in-page wizard so that's a bit fuzzy which guidelines to follow. But also it's not so important.

Regardless, we should avoid using hardcoded pixels in CSS stylesheets. Instead we need to rely on the paddings of the existing elements. Removing the Bullseye and keeping the rest of elements gives top padding 56px which is fine I believe.

It's slightly less than the 64px suggestion for in-page empty states, and ,more than the 24px suggestion for the in-wizard empty states. But since we actually have a wizard header, I think it's an acceptable padding for our case.

Reply:
Ok, I removed the hard-coded padding.